community/volunteers.md

```diff
@@ -0,0 +1,16 @@
+This page lists members of the SPIFFE community who have signalled a willigness to represent the SPIFFE project at various public fora, which may include public events (such as KubeCon maintainers talks or conference booths), virtual events (such as CNCF Webinars), or other events.
+
+Some notes:
+* *If you are coordinating an event and looking for a speaker or other support on SPIFFE, feel free to reach out to any of those listed below. Inclusion on this list is for the convienence of event organizers, and does not imply a commitment by any of the speakers to participate in any particular event.*
+*  *While we strive to ensure all speakers listed below respect and closely follow our [Code of Conduct](../CODE-OF-CONDUCT.md), and will remove those that do not, inclusion in this list does not represent an endorsement of that individual from the SPIFFE community. Please conduct your own due diligence before inviting anyone from this list to an event.*
+
+| Who | Contact | Location | Preferred speaking topics | Preferred duties |
```

Comment:
```suggestion
| Who | Contact | Location | Preferred speaking topics | Preferred duties | Volunteer Notes
```

additional column may be helpful here on other notes from the volunteer; mine would be `Requires additional lead time for review and approval by employer`